src/App.js

```diff
@@ -1,29 +0,0 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, please consider restoring this file as it is the entry point of your app. According to the requirements:

![image](https://user-images.githubusercontent.com/77069011/220457089-6031ba13-2e7e-49d1-95a4-1e543dbd814f.png)

So please consider doing something like this:

```js
const App = () => {
  return <CalculatorContainer />
}
```

This structure will come in handy when you are required to add routes, so please apply it! 🤓 🔧